mlos_core/setup.py

```diff
@@ -28,6 +28,7 @@
 extra_requires: Dict[str, List[str]] = {
     'emukit': ['emukit'],
     'skopt': ['scikit-optimize<=0.9.0'],  # FIXME: temporarily work around some version mismatch issues (PR 850)
+    'flaml': ['flaml[blendsearch]'],
```

Comment:
Are there other suboptions that might make sense here?

Reply:
I don't think so. Here's the full list of suboptions: https://microsoft.github.io/FLAML/docs/Installation

Most of these suboptions are used for AutoML purposes. Blendsearch is the only that makes sense for our cases imho.